packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "3d4a8d7caec7cd0239dbc40e39c8ec4d965a9a67",
-    "ref_origin" : "dcos-mesos-1.0.3-rc1"
+    "ref": "0fd32e4e537507f900a5c3ea0bf3051bea76b0d3",
+    "ref_origin" : "dcos-mesos-1.0.x-17ac024"
```

Comment:
Looks like you updated `dcos-mesos-1.0.x-17ac024` to be based on `4154f66`, just like `dcos-mesos-1.0.x-4154f66` but the buildinfo.json still points to the wrong branch. Please fix.

Please also rebase/update mesosphere/dcos-enterprise#841 when you update this PR, so we can see the EE tests run again (and pass the build this time).